hack/e2e.go

```diff
@@ -91,76 +91,184 @@ func main() {
 	if *build {
 		// The build-release script needs stdin to ask the user whether
 		// it's OK to download the docker image.
-		cmd := exec.Command(path.Join(*root, "hack/e2e-internal/build-release.sh"))
+		cmd := exec.Command("make", "quick-release")
```

Comment:
We don't always want to do a quick release. On post-submit Jenkins we want to do the full release.